config/subjects.yaml

```diff
@@ -0,0 +1,7 @@
+subjects:
+  name:
+    - 'apple'
```

Comment:
This list should be updated to: [ref](https://notch8.slack.com/archives/C08ADGYKLG1/p1740755864522189?thread_ts=1740591200.867569&cid=C08ADGYKLG1)

·              Applied Science
·              Arts and Humanities
·              Business and Communication
·              Career and Technical Education
·              Education
·              Engineering
·              English Language Arts
·              Health Sciences
·              History
·              Law
·              Life Science
·              Mathematics
·              Physical Science
·              Social Science

Reply:
We should prob also update the seeds script that Kirk recently made

Reply:
> We should prob also update the seeds script that Kirk recently made

lol I just asked that in slack!